src/transformers/models/code_llama/tokenization_code_llama_fast.py

```diff
@@ -20,11 +20,8 @@
 
 from ...tokenization_utils_fast import PreTrainedTokenizerFast
 from ...utils import is_sentencepiece_available, logging
-from ...utils.versions import require_version
 
 
-require_version("tokenizers>=0.13.3")
```

Comment:
Do you intent to remove this one?

Reply:
yes because ``tokenizers>=0.21,<0.22`` in setup.py